docker-compose_production.yml

```diff
@@ -260,6 +272,13 @@ volumes:
       device: ${DOCKER_MIGRID_ROOT}/state
       o: bind
   
+  state-ro:
```

Comment:
I would choose another name cause this is not the state directory but the vgrid_files dir.

Reply:
@benibr i don't have any good ideas to what else it should be named. All the dirs are in the migrid_root directory, and this also plays with the state dir.